src/Algebra/Graph/ToGraph.hs

```diff
@@ -266,7 +267,7 @@ class ToGraph t where
     -- @
     -- topSort == Algebra.Graph.AdjacencyMap.'AM.topSort' . toAdjacencyMap
     -- @
-    topSort :: Ord (ToVertex t) => t -> Maybe [ToVertex t]
+    topSort :: Ord (ToVertex t) => t -> Either (NonEmpty (ToVertex t)) [ToVertex t]
```

Comment:
I guess you should be able to reuse `AM.Cycle` here as well.

Reply:
Actually, the documentation above is outdated, it should be:

`topSort == Algebra.Graph.AdjacencyMap.Algorithm.'AM.topSort' . toAdjacencyMap`

It would be great if you fix this and other similar problems (if any) in this module. Feel free to do this in this PR.

Reply:
Will do. Would it matter that `AM.Cycle` and `AIM.Cycle` both define the synonym?

Reply:
I think it's fine if both modules define their own synonyms.